aggregator/pom.xml

```diff
@@ -47,6 +47,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-messaging-kafka</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.flywaydb</groupId>
+            <artifactId>flyway-database-postgresql</artifactId>
+            <version>10.12.0</version>
```

Comment:
This dependency is listed in the [Quarkus BOM](https://github.com/quarkusio/quarkus/blob/ecc068192b437f8a001faafb0cff252c18be0fb4/bom/application/pom.xml#L5709-L5713), so we shouldn't override the version here.
```suggestion
```